doc/develop/test/twister.rst

```diff
@@ -149,7 +149,23 @@ name:
 type:
   Type of the board or configuration, currently we support 2 types: mcu, qemu
 simulation:
-  Simulator used to simulate the platform, e.g. qemu.
+  Simulator(s) used to simulate the platform, e.g. qemu.
+
+  .. code-block:: yaml
+
+      simulation:
+        - name: qemu
+        - name: armfvp
```

Comment:
Any further guidance/examples on how to use multiple simulators for a single board? So far for ARC, we have different "boards" defined for nSIM simulations (under boards/snps/nsim/... and qemu simulations (boards/qemu/arc/...).
Is the idea to get rid of the boards/qemu and move them under the vendor/architecture instead?

Reply:
Not familiar with those.

I'd say that if you can build the very same binary and run it on different simulator, then they could be brought together under a single board and list the usable simulators.